packages/web-components/src/components/cbp-toast/cbp-toast.tsx

```diff
@@ -47,9 +59,9 @@ export class CbpToast {
             />
```

Comment:
This is still being rendered instead of slotted? Let's discuss.

Reply:
updated icon to slotted and set in story as discussed in call